models/preferencecategory.js

```diff
@@ -12,21 +10,25 @@ module.exports = (sequelize, DataTypes) => {
     static associate(models) {
       // define association here
     }
-  };
-  PreferenceCategory.init({
-    name: {
-      allowNull: false,
-      unique: true,
-      type: DataTypes.STRING,
-      validate: {
-        max: 32,
-        is: ["[a-z]",'i']
-      }
+  }
+  PreferenceCategory.init(
+    {
+      name: {
+        allowNull: false,
+        unique: true,
+        type: DataTypes.STRING,
+        validate: {
+          max: 1000,
+          //isNumeric: false, // ask patrice why this validation is failing
```

Comment:
Not sure what you mean by this comment. The name of a preference category should really only be letters. 'Programming Languages', 'Skills', 'Platforms', etc, etc. As long as it's not null or a blank string `''`, then we're sweet